### DISCUSSION TITLE
Welcome to Noisedeck Discussions!

### DISCUSSION THREAD

**Discussion by jesshewitt** [noisedeck/noisedeck-docs#1] [General]
## 👋 Welcome!
  We’ve created this forum as a place for our community to connect. Feel free to:
  * Introduce yourself! Let us know who you are and how you're using Noisedeck
  * Share works you have created with Noisedeck
  * Engage with other community members
  * Ask questions, suggest features, or report issues

You'll need a free GitHub account to post here.

Please adhere to [Github's Community guidelines](https://docs.github.com/articles/github-community-guidelines) and be excellent to each other. 🥰

**Comment by VegaDeftwing** [comment]
Hey there :wave:  I imagine both of you have already met me on Twitter after that long exchange (I'm Vega_DW over there) thought I'd chime in here in case you want to ping me on any of the development notes and feature requests I talked about- At the very least this is certainly a bit more organized than that Twitter chat.

Also I just realized I've actually spent far to many hours looking at Alex Ayars work before, as I went though literally every preset in py-noisemaker at one point. Now I'm 100x more excited to see where Noisedeck goes :beers:

**Reply by jesshewitt** [reply]
Hey, good to see you here! Alex and I both use the Noisedeck Twitter account, but I'm the one you were speaking with regarding the development notes/feature requests. 👋

**Comment by morganite-proto** [comment]
HI! I'm Morganite. I'm an IT student by day but by night I do stuff in virtual reality, mainly social apps like VRChat. There's an underground music and DJ scene in the VR world, and I believe Noisedeck can be a fantastic tool for visuals. I'm also on Twitter as @MorganiteProto and my music twitter is @MorganiteNoises. Excited to see how the program develops! 💖

**Reply by jesshewitt** [reply]
Hi, welcome! Glad you're enjoying Noisedeck. We are definitely interested in any feedback/suggestions/feature requests you may have for visuals in VR.

**Comment by ShamanicArts** [comment]
Hi ! I’m Shamanic a mixed media artist , worked mostly in music and performance, as a singer beat boxer and frontman, especially enjoy improvisation. Currently experimenting with learning to code, JavaScript and Python, and visual arts through Hydra, touchdesigner and this platform. Mostly use noise deck to generate assets which I then use to create other pieces of art.

**Reply by jesshewitt** [reply]
Hi Shamanic, welcome! I'm very interested to see what you're creating.
